File: Day - 2/question3.c
#include <stdio.h>
int main(){
    /*Write a program to Find product of digits. */
    int n , d , pro = 1;
    printf("enter the nuber n=");
    scanf("%d", &n);
    while(n>0){
        d = n%10;
        pro= pro*d;
        n = n/10;
    }
    printf("product of a given number is %d", pro);
    return 0;
}

File: Day - 16/Question -3.py
# Write a program to Find pair with given sum.
n = int(input("enter number of elements = "))

array = []
for i in range(n):
    array.append(int(input()))
target = int(input("enter the target sum = "))

found = False
for i in range(n):
    for j in range(i + 1, n):
        if array[i] + array[j] == target:
            print("Pair found:", array[i], array[j])
            found = True
if not found:
    print("No pair found")
